hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/file/TestOMFileCreateRequest.java

```diff
@@ -146,7 +141,7 @@ public void testValidateAndUpdateCache() throws Exception {
 
     // Check open table whether key is added or not.
 
-    omKeyInfo = omMetadataManager.getOpenKeyTable().get(openKey);
+    omKeyInfo = verifyPathInOpenKeyTable(keyName, id, true);
     Assert.assertNotNull(omKeyInfo);
```

Comment:
This check can be removed as we already check this in verifyPathInOpenKeyTable.

Reply:
Done in latest commit